CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.21.1](https://github.com/aws/aws-cdk/compare/v1.21.0...v1.21.1) (2020-01-16)
+
```

Comment:
Changelog should probably contain SOMETHING

Reply:
Yeah. It’s because your commitS didn’t use conventional commits. Feel free to add by hand.